src/test/scala/org/broadinstitute/dsde/firecloud/webservice/SubmissionApiServiceNegativeSpec.scala

```diff
@@ -48,15 +48,14 @@ final class SubmissionApiServiceNegativeSpec extends ServiceSpec with Submission
     }
 
     "non-POST requests hitting the /workspaces/*/*/submissions/validate path are not passed through" in {
-      // Also excluding DELETE and GET since the path matches
       // /workspaces/*/*/submissions/{submissionId} APIs as well
-      allHttpMethodsExcept(DELETE, GET, POST) foreach { method =>
+      allHttpMethodsExcept(POST) foreach { method =>
         checkIfPassedThrough(submissionServiceRoutes, method, s"$localSubmissionsPath/validate", toBeHandled = false)
       }
     }
 
-    "non-DELETE/GET requests hitting the /workspaces/*/*/submissions/* path are not passed through" in {
-      allHttpMethodsExcept(DELETE, GET) foreach { method =>
+    "non-DELETE/GET/PATCH requests hitting the /workspaces/*/*/submissions/* path are not passed through" in {
```

Comment:
Since we're adding a new endpoint (rather than modifying an existing one), we need an OK from security/compliance before merging. I've followed up on our previous request for a compliance review in the ticket here: https://broadworkbench.atlassian.net/browse/BW-761?focusedCommentId=50246